src/components/radioInput/radioInput.tsx

```diff
@@ -8,27 +8,61 @@ interface RadioInputProps {
   name: string;
   label: string;
   className?: string;
+  checked?: boolean;
+  disabled?: boolean;
+  variant?: 'simpleOption' | 'buttonOption';
   onChange?: React.ChangeEventHandler<HTMLInputElement>;
 }
 export const RadioInput = React.forwardRef<HTMLInputElement, RadioInputProps>(
-  function RadioInput({ value, name, label, className, onChange }, ref) {
+  function RadioInput(
+    {
+      value,
+      name,
+      label,
+      className,
+      checked,
+      disabled,
+      variant = 'simpleOption',
+      onChange,
+    },
+    ref,
+  ) {
     const ContClass = classNames(classes.cont, className);
 
-    return (
-      <div className={ContClass}>
-        <input
-          name={name}
-          ref={ref}
-          className={classes.input}
-          type="radio"
-          value={value}
-          id={value}
-          onChange={onChange}
-        />
-        <label className={classes.name} htmlFor={value}>
-          {label}
+    if (variant === 'simpleOption')
```

Comment:
What if there will be another variant in future? It's not a good approach to modify existing code. Follow the Open/Closed principle (entity should be open for extension but closed for modification).

If I were, I would create another component for every input or use composition here (e.g. pass classNames or/and label component)